tools/online_editor/src/index.ts

```diff
@@ -73,6 +73,7 @@ import slint_init, * as slint from "@preview/slint_wasm_interpreter.js";
 
   let base_url = "";
   let event_loop_started = false;
+  let current_preview_instance: slint.WrappedInstance | null = null;
```

Comment:
`let current_preview_instance: slint.WrappedInstance | undefined;`

Reply:
Why choose `undefined` as option instead of `null`? It is either an object or null, but not an undefined value.

Reply:
It does not make much of a difference, you just need to type a few more letters.

Reply:
If that's the only argument then I very much prefer using "null". Null is defined to represent the intentional absence of an object value, which is exactly what it is here.